src/seedu/addressbook/parser/ExamField.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.parser;
+
+/**
+ * Enum used to indicate which exam field a string belongs to
+ * */
+public enum ExamField {
+    examName,
+    subjectName,
+    examDate,
+    examStartTime,
+    examEndTime,
+    examDetails,
+    isPrivate
+}
```

Comment:
Actually, this could be part of Exams if ya don't want to create a whole new class for it

Reply:
Fair enough

Reply:
As clarified in:
https://github.com/nusCS2113-AY1819S1/forum/issues/61